src/appshell/view/preferences/preferencesmodel.cpp

```diff
@@ -146,7 +146,7 @@ void PreferencesModel::load(const QString& currentPageId)
     m_rootItem = new PreferencePageItem();
 
     QList<PreferencePageItem*> generalItems {
-        makeItem("general-start", qtrc("appshell", "Programme start"), IconCode::Code::NONE,
+        makeItem("general-start", qtrc("appshell", "Programme Start"), IconCode::Code::NONE,
```

Comment:
`"Program start"`, US spelling and title case

Reply:
Converted to `"Program start"`
Fixed :)